src/polymer/iron-flex-layout-import.ts

```diff
@@ -59,8 +59,9 @@ class IronFlexLayoutImport extends HtmlRule {
           parsedDocument.sourceRangeForAttributeValue(imp, 'href')!;
       const warning = new FixableWarning({
         code: 'iron-flex-layout-import',
-        message: `${href} import is deprecated. ` +
-            `Replace with ${correctImport} import.`,
+        message: `${href} import is deprecated in iron-flex-layout v1, and not shipped in iron-flex-layout v2.x.
+Replace it with ${correctImport} import.
+Run \`iron-flex-layout-classes\` to include the required style modules.`,
```

Comment:
Recommend the wording:

Run the lint rule  \`iron-flex-layout-classes\` with `--fix` to include...

Reply:
Done 👍